runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -870,7 +877,15 @@ func s:DeleteCommands()
     if empty(s:k_map_saved)
       nunmap K
     else
-      call mapset('n', 0, s:k_map_saved)
+      " call mapset('n', 0, s:k_map_saved)
+      let mode = s:k_map_saved.mode !=# ' ' ? s:k_map_saved.mode : ''
+      call nvim_set_keymap('n', 'K', s:k_map_saved.rhs, {
+        \ 'expr': s:k_map_saved.expr ? v:true : v:false,
+        \ 'noremap': s:k_map_saved.noremap ? v:true : v:false,
+        \ 'nowait': s:k_map_saved.nowait ? v:true : v:false,
+        \ 'script': s:k_map_saved.script ? v:true : v:false,
+        \ 'silent': s:k_map_saved.silent ? v:true : v:false,
```

Comment:
You can just pass `expr` etc. directly; the fields always exist and have the correct truthiness in vimscript.

Reply:
`nvim_set_keymap` doesn't like that unfortunately, and throws an error that it expected a boolean and got a number. Even `==` operator doesn't work.